pkg/kube-metrics/server.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubemetrics
+
+import (
+	"fmt"
+	logg "log"
```

Comment:
Any reason we're using the standard log pkg vs logr like elsewhere:
```Go
var log = logf.Log.WithName("kubemetrics")
```
`ServeCRMetrics()` is using the same.

Reply:
oops yes, will fix that, nicely spotted, pretty sure I was testing this as a standalone and it was leftover!